packages/terra-abstract-modal/src/terra-dev-site/doc/example/AbstractModalCloseOnOutsideClick.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import AbstractModal from 'terra-abstract-modal/lib/AbstractModal';
+import styles from './ExampleAbstractSize.scss';
+
+class AbstractModalCloseOnOutsideClick extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      isOpen: false,
+    };
+
+    this.handleOpenModal = this.handleOpenModal.bind(this);
+    this.handleCloseModal = this.handleCloseModal.bind(this);
+  }
+
+  handleOpenModal() {
+    this.setState({ isOpen: true });
+  }
+
+  handleCloseModal() {
+    this.setState({ isOpen: false });
+  }
+
+  render() {
+    return (
+      <div>
+        <AbstractModal
+          ariaLabel="Modal disable close on outside click"
+          isOpen={this.state.isOpen}
+          closeOnOutsideClick={false}
+          onRequestClose={this.handleCloseModal}
+          classNameModal={styles['fixed-size']}
+        >
+          <div style={{ height: '100%', width: '100%', overflow: 'hidden', backgroundColor: 'lightgreen' }}>
+            <h1>Modal disable close on outside click</h1>
+            <br />
```

Comment:
Are these br's necessary for this example? It looks like they are used just for spacing, in which it would be better just to do styles on the paragraphs.

Reply:
These were present in the previous examples, we just moved their location in this PR.